packages/splitter/README.md

```diff
@@ -60,26 +66,28 @@ const Splitter = () => {
 import {
   SplitterContainer,
   SplitterType,
-  SplitterOrientation
+  SplitterOrientation,
+  SplitterPlacement
```

Comment:
same re: `SplitterPosition`.

Reply:
Fixed in https://github.com/zendeskgarden/react-containers/pull/405/commits/72feeb0d9c03ec822bb561b57da6c3adc01a7996